### DISCUSSION TITLE
About the size of the cylinder and sphere

### DISCUSSION THREAD

**Discussion by mar-stamp** [merkourisa/Augmented-Reality#11] [Q&A]
I have successfully made the code for both the cylinder and sphere but no matter what I'm trying to do the size remains the same. Both the cylinder and the sphere are way too big in comparison with the cube. Does anyone have any idea why this happens?

Here is my code:

`<a-entity id="box" geometry="primitive: box; width: 0.2; depth: 0.2; height: 0.2" position="0 0 0" material="color: #4CC3D9"></a-entity>
            <a-cylinder id="cylinder" geometry="primitive: cylinder; radius= 0.2; height=0.2" position="1 1 1" material="color: #FFC65D"></a-cylinder>
            <a-sphere id="sphere" geometry="primitive: sphere; radius= 0.2;" position="-1 -1 -1" material="color: #EF2D5E"></a-sphere>`

**Comment by sdoulaveris** [comment]
ti einai edw

**Comment by Costas-Lemonis** [comment]
Ο κύλινδρος θέλει τις εντολές radius και height για να ορίσεις το μέγεθος του, ενώ η σφαίρα μόνο radius

**Reply by mar-stamp** [reply]
It doesn't work. I have uploaded my code in my first question.

**Reply by Costas-Lemonis** [reply]
Άλλαξε στον κώδικα σου τα
radius= 0.2 σε radius: 0.2 
height=  0.2 σε height: 0.2

**Comment by merkourisa** [comment]
@mar-stamp Should be an 

`<a-cylinder id="cylinder" geometry="primitive: cylinder; radius= 0.2; height=0.2" position="1 1 1" material="color: #FFC65D"></a-cylinder>`

or  an

`<a-entity id="cylinder" geometry="primitive: cylinder; radius= 0.2; height=0.2" position="1 1 1" material="color: #FFC65D"></a-entity>`  ?
